### PR TITLE
Chapter 11: Exceptions

### PR DESCRIPTION
**Pull request by speudusa**

### CODE REVIEW

File: content/exceptions/handling-exceptions/index.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+When you include exceptions in your C# programs, you must decide what should take place once one is thrown. Some languages, like Java, require exceptions to be handled at the time of compiling. These are **checked exceptions**. When an exception is not handled in compiling, it is passed to runtime and called an **unchecked exception**. All exceptions in C# are unchecked. Therefore it is up to you, the programmer, to decide what to do to handle one when the need arises in runtime.
+
+Here’s some advice to consider when contemplating when to handle an exception. This comes from Karl Seguin’s [Foundations of Programming](https://www.openmymind.net/FoundationsOfProgramming.pdf).
+
+   1. Only handle exceptions that you can actually do something about.
+
+   1. You can’t do anything about the vast majority of exceptions.
```
Comment (codinglikeagirl42) [inline content/exceptions/handling-exceptions/index.md:21]:
```suggestion
   1. Only handle exceptions that you can actually do something about.
   1. You can’t do anything about the vast majority of exceptions.
```

**Review by codinglikeagirl42 — DISMISSED** [review]
Please update the metadata on all the pages.  Also need a Reading subfolder and move Introduction to Exceptions, Handling Exceptions, and Common Exception Objects folders inside.

**Comment by speudusa** [comment]
fixed the file tree.  updated the metadata.  updated the exercises. :)

**Review by codinglikeagirl42 — COMMENTED** [review]
Looks good, merge away!

**Pull request merged**